oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/ModelColumn.java

```diff
@@ -30,19 +31,21 @@
     private final boolean matchQuery;
     private final boolean storageOnly;
     private final int length;
+    private final Column.AnalyzerType analyzer;
 
     public ModelColumn(ColumnName columnName,
                        Class<?> type,
                        Type genericType,
                        boolean matchQuery,
                        boolean storageOnly,
                        boolean isValue,
-                       int length) {
+                       int length, final Column.AnalyzerType analyzer) {
```

Comment:
Please use a new line to follow the format. And make all parameters `as final` or none. Don't mix the style